### DISCUSSION TITLE
can't read properties of fetched data

### DISCUSSION THREAD

**Discussion by nikstaaar** [neuefische/questions#189] [Web] [answered]
Hi, 
i have built an array of objects from several api pages with the promise.all function.
i can log out each individual object in this array with a map function but when I want to get only specific properties from the objects I get the "Cannot read properties of null" error. I have looked several times over the objects to make sure I didn't make a mistake with the keys. I would appreciate your help :)

https://github.com/nikstaaar/capstone-project/pull/11

**Comment by actyralf** [comment, marked as answer]
It seems that the array returned from the api contains some `null` values. you could filter out these before mapping the array.
